### PR TITLE
[sdk] adds cryptography flag, fixes address generation

### PR DESCRIPTION
**Pull request by damirka**

### CODE REVIEW

**Review by 666lcz — APPROVED** [review]

File: sdk/typescript/experimental/src/sui-client.ts [hunk 1]
```diff
@@ -267,5 +268,5 @@ export class SuiClient {
  * Get the Sui address from a keypair using a `sha3_256` hash substring.
  */
 export function getAddress(publicKey: Uint8Array): string {
-  return sha3_256(publicKey).slice(0, 40);
+  return sha3_256([0, ...Array.from(publicKey)]).slice(0, 40);
```
[the remark this answers is not in the document]
Comment (damirka) [reply sdk/typescript/experimental/src/sui-client.ts:271]: It's a temp measure to unblock my progress with dapps. Ideally we'd want a cryptography interface, nacl won't be the only one it seems.

**Comment by joyqvq** [comment]
opened an issue to track, otherwise lgtm

**Review by joyqvq — APPROVED** [review]

**Comment by punwai** [comment]
LGTM!

**Review by punwai — APPROVED** [review]

**Comment by damirka** [comment]
Closing this buddy as experimental sdk is being reworked.

**Pull request closed without merging**